api/python/quilt3/api.py

```diff
@@ -182,6 +212,15 @@ def config(*catalog_url, **config_values):
     # Return current config
     return QuiltConfig(CONFIG_PATH, local_config)
 
+
+@ApiTelemetry("api.disable_telemetry")
+def disable_telemetry():
+    """ Permanently disable sending of anonymous usage metrics """
+    _config(telemetry_disabled=True)
```

Comment:
This doesn't disable anything for the current session, though.

Reply:
Good catch! I added a somewhat hacky workaround for that in `telemetry.py` which should solve this for now